Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/RideRequestUtils.java

```diff
@@ -317,13 +317,6 @@ public static  JSONObject getZoneConfig(String tag, Context context){
     public static void setSpecialZoneAttrs(SheetAdapter.SheetViewHolder holder, String specialLocationTag, Context context) {
         try{
             JSONObject zoneConfig = getZoneConfig(specialLocationTag,context);
-            holder.specialLocationTag.setCardBackgroundColor(Color.parseColor(zoneConfig.get("backgroundColor").toString()));
```

Comment:
Why is this removed?
If not required then remove it from view as well
And remove all properties related to it from `Model` and `Adapter`